### DISCUSSION TITLE
Delayed query of feeds

### DISCUSSION THREAD

**Discussion by OrkoGrayskull** [FreshRSS/FreshRSS#7124] [Q&A]
### Describe the bug

First of all, this is not necessarily a bug, but none of the other categories fit.

I use my own server and have about 25 RSS feeds. I use the cron command to update them:

`/20 * * * * * php /path/to/FreshRSS/app/actualize_script.php > /tmp/FreshRSS.log 2>&1`

This means that the feeds are updated every 20 minutes (e.g. 19:00, 19:20 and 19:40). Although some of the feeds have new content, this is not picked up by FreshRSS. This is especially true for YouTube, where delays of between 1 and 2 hours are unfortunately normal. My IP address or that of the server and also the user agent is not blocked by the remote station. My question is therefore, what can I do to ensure that new content is fetched promptly and reliably?

### To Reproduce

Nothing.

### Expected behavior

Timely and reliable retrieval of new RSS content.

### FreshRSS version

1.25.0-dev

### Environment information

- Database version: mariadb-server (1:10.11.6-0+deb12u1)
- PHP version: 8.2
- Installation type: git
-Web server type: nginx
- Device: Mobile / PC
- OS: Android 15, Debian
- Browser: Firefox, Chromium

**Comment by math-GH** [comment]
The default is 1h. You can change it till 20min.

![grafik](https://github.com/user-attachments/assets/00a8168d-f9dd-4c30-a413-bc180890d544)

It is possible to set it down to 15min. for the feeds in the feed setting.
![grafik](https://github.com/user-attachments/assets/68679e0b-a5c7-49ee-87b0-cfa6ea2929c8)

**Comment by OrkoGrayskull** [comment]
Now I'm a bit confused, I thought the cron job updates the feeds. Why is there another setting under Archiving?

**Comment by Alkarex** [comment]
This is to allow feeds to have a lower refresh frequency than the cron, which you can run as often as you want.
High feed refresh frequency is wasting resources and risks leading to your IP being blocked (or worse, the user-agent, which affects other users).
For receiving articles instantly, there is the WebSub protocol, which FreshRSS supports, and which is used by some feeds. You can test that your instance is correctly set up with http://push-tester.cweiske.de/
